rpcs3/rpcs3qt/custom_table_widget_item.h

```diff
@@ -26,12 +26,17 @@ class custom_table_widget_item : public QTableWidgetItem
 		}
 	}
 
-	bool operator <(const QTableWidgetItem &other) const
+	bool operator <(const QTableWidgetItem &other) const override
 	{
 		return data(m_sort_role) < other.data(m_sort_role);
 	}
 
-	void setData(int role, const QVariant &value, bool assign_sort_role = false)
+	void setData(int role, const QVariant &value) override
```

Comment:
You can just drop ths entire override function... All you had to do was remove the default value I guess

Reply:
Doing that causes errors.